Findscript.cmake.in

```diff
@@ -1,11 +1,11 @@
-# - Find an ovsrpro installation.
-# ovsrpro_DIR
+# - Find an @CMAKE_PROJECT_NAME@ installation.
+# @CMAKE_PROJECT_NAME@_DIR
```

Comment:
(question) is there a reason for ovsrpro to have it's own version of `Findscript.cmake.in`? all our other `*pro` repos (internpro and webpro) don't need to do this - they use the Findscript.cmake.in from externpro...

if you can help me identify what you want to customize then perhaps I can add these things to the next release of externpro...

Reply:
@smanders the reason we assumed we need a version here is that it does not have the same release cycle as externpro. so we could use externpro 17.05.1 (just example) and ovspro 17.10.2. Is there a better way?

Reply:
I'm not seeing how the release cycle has anything to do with this file...
* ovsrpro specifies the release of externpro it's using
  * https://github.com/distributePro/ovsrpro/blob/master/CMakeLists.txt#L4
* the `Findscript.cmake.in` from that release of externpro would be used to generate the `Findovsrpro.cmake` file that's in the ovsrpro installer and copied into projects that require (want to find) ovsrpro...

what am I missing? perhaps you could explain the release cycle thought? are you wanting to update the `Findovsrpro.cmake` to support a new release of Visual Studio, for example, without specifying the release of externpro that supports the new release of Visual Studio?

Reply:
Using your example (use externpro 17.05.1 and ovsrpro 17.10.2)... are you saying that there need to be additional things in `Findovsrpro.cmake` 17.10.2 that aren't in the `Findscript.cmake.in` from externpro 17.05.1? again - I think we need to identify what things in `Findscript.cmake.in` that you want to customize

Reply:
@smanders we don't really need to customize stuff in here. we think/thought we needed it to specify install location (old to /opt/ovsrpro but new looks like /opt/extern/ovspro. right?), to specify version and to find the ovsrpro libraries. if all that is not needed, then i am for removing this.

Reply:
cmake looks in `/opt/extern` for an externpro or internpro or webpro or ovsrpro install... (it used to look in `/opt/externpro` for an externpro or internpro or webpro install)

install command and directory listing:
```bash
$ sudo ./externpro-18.01.1-gcc630-64-Linux.sh --prefix=/opt/extern --include-subdirs
$ ls -1 /opt/extern
externpro-18.01.1-gcc630-64-Linux
externpro-18.01.1-gcc720-64-Linux
internpro-18.01.2-gcc630-64-Linux
webpro-18.01.1-gcc630-64-Linux
webpro-SL18.01.1-gcc630-64-Linux
```